infrastructure/local/network_docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
```

Comment:
After making separate compose file for the networks, I haven't seem to be running into the
- #1003

Reply:
Oh no, I got it first try :disappointed: 
```
network local_rafiki declared as external, but could not be found
```

Reply:
:confused:  I'm only getting that error, even after pruning networks
```
docker network prune
```

Reply:
Hmm, I must have just had it created previously. What do you think about combining docker-compose and peer-docker-compose files?

Reply:
That's probably the most straightforward way to prevent #1003

Reply:
Would you ever run just a single instance, for example?

Reply:
Ok, I pushed to merge the peer docker compose file into the main one, let me know if that works

Reply:
I think we should support running a single rafiki